frontend/src/metabase/query_builder/components/QueryHeader.jsx

```diff
@@ -289,7 +289,6 @@ export default class QueryHeader extends Component {
 
         buttonSections.push([
           <ModalWithTrigger
-            ref="move"
             key="move"
             full
```

Comment:
We should remove the full here to keep the experience for moving items the same throughout the product.